platform/persistence/platform-persistence-commands/src/main/java/org/codice/ddf/persistence/commands/StoreImportCommand.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.persistence.commands;
+
+import static org.codice.ddf.persistence.PersistentItem.BINARY_SUFFIX;
+import static org.codice.ddf.persistence.PersistentItem.DATE_SUFFIX;
+import static org.codice.ddf.persistence.PersistentItem.INT_SUFFIX;
+import static org.codice.ddf.persistence.PersistentItem.LONG_SUFFIX;
+import static org.codice.ddf.persistence.PersistentItem.SUFFIXES;
+import static org.codice.ddf.persistence.PersistentItem.TEXT_SUFFIX;
+import static org.codice.ddf.persistence.PersistentItem.XML_SUFFIX;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonIOException;
+import com.google.gson.JsonSyntaxException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.FileVisitOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+import org.apache.commons.lang.StringUtils;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Completion;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.apache.karaf.shell.support.completers.FileCompleter;
+import org.codice.ddf.persistence.PersistenceException;
+
+@Service
+@Command(
+  scope = "store",
+  name = "import",
+  description = "Import entries into the persistent store."
+)
+public class StoreImportCommand extends AbstractStoreCommand {
+
+  @Argument(
+    name = "File path or Directory path",
+    description =
+        "Path to a file or a directory of file(s) to be ingested. Paths can be absolute or relative to installation directory.",
+    index = 0,
+    multiValued = false,
+    required = true
+  )
+  @Completion(FileCompleter.class)
+  String filePath;
+
+  private final Gson gson = new Gson();
+  private final SimpleDateFormat formatter = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz");
```

Comment:
✏️ Might be nice to expose this date format (package private) in `StoreExportCommand` and import it here so they don't get out of sync